chia/wallet/util/debug_spend_bundle.py

```diff
@@ -65,7 +65,6 @@ def debug_spend_bundle(spend_bundle, agg_sig_additional_data=bytes([3] * 32)) ->
             print("*** BAD PUZZLE REVEAL")
             print(f"{puzzle_reveal.get_tree_hash().hex()} vs {coin_solution.coin.puzzle_hash.hex()}")
             print("*" * 80)
-            breakpoint()
```

Comment:
Was this intentional?

Reply:
yeah that's the `debug-statements` hook which makes sure there are is no debug stuff commited